crates/ruff/src/rules/flake8_pie/rules/unnecessary_range_start.rs

```diff
@@ -0,0 +1,78 @@
+use num_bigint::BigInt;
+use ruff_python_ast::{self as ast, Constant, Expr, Ranged};
+
+use ruff_diagnostics::Diagnostic;
+use ruff_diagnostics::{AlwaysAutofixableViolation, Fix};
+use ruff_macros::{derive_message_formats, violation};
+
+use crate::autofix::edits::{remove_argument, Parentheses};
+use crate::checkers::ast::Checker;
+use crate::registry::AsRule;
+
+/// ## What it does
+/// Checks for `range` calls with an unnecessary `start` argument.
+///
+/// ## Why is this bad?
+/// `range(0, x)` is equivalent to `range(x)`. The `start` argument is unnecessary.
+///
+/// ## Example
+/// ```python
+/// range(0, 3)
+/// ```
+///
+/// Use instead:
+/// ```python
+/// range(3)
+/// ```
```

Comment:
```suggestion
/// ```
///
/// ## References
/// - [Python documentation: `range`](https://docs.python.org/3/library/stdtypes.html#range)
```

Think it could be worth linking to the documentation, so people can read how `range` is defined.